cdk/package.json

```diff
@@ -12,15 +12,15 @@
     "diff": "cdk diff --path-metadata false --version-reporting false"
   },
   "devDependencies": {
-    "@guardian/cdk": "59.5.2",
+    "@guardian/cdk": "59.5.3",
```

Comment:
This version of CDK is broken! See https://github.com/guardian/cdk/pull/2484

Reply:
I'll revert!

Reply:
Updated 😄